docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -49,8 +49,9 @@ and time availability), claim it by following these steps:
 * Log into your account, if you haven't already, by clicking "GitHub Login"
   or "DjangoProject Login" in the upper left of the ticket page.
 
-* Claim the ticket by clicking the "assign to myself" radio button under
-  "Action" near the bottom of the page, then click "Submit changes."
+* Claim the ticket by clicking the "Modify Ticket" button near the bottom
+  of the page. Then click the "assign to" radio button under
+  "Action", next to your username. Finally click "Submit changes."
```

Comment:
I like it thank you @mariahynes 

I was thinking we could move the modify ticket mention to the logged in bullet and then put the submit in a separate bullet, something like this:
```
* Log into your account, if you haven't already, by clicking "GitHub Login"
  or "DjangoProject Login" in the upper left of the ticket page. Once logged
  in, you can then click the "Modify Ticket" button near the bottom of the
  page.

* Claim the ticket by clicking the "assign to" radio button in the "Action"
  section. Your username will be filled in the textbox by default.

* Finally, click the "Submit changes" button at the bottom to save.
```

Reply:
Yes, that's neater. I'll update 👍

Reply:
That is updated now Sarah:

BEFORE:
>![Instructions with two bullet points](https://github.com/user-attachments/assets/be0734ae-75da-4242-80dc-5113b97be8e8)

AFTER:
>![Instructions reformatted into three bullet points](https://github.com/user-attachments/assets/d835b5b3-0214-44e9-8fb0-5cc9e973acb7)